lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -486,7 +481,7 @@ static lldb::ThreadPlanSP GetStepThroughTrampolinePlan(Thread &thread,
         log->Printf("Stepped to thunk \"%s\" (kind: %s) but could not "
                     "find the thunk target. ",
                     symbol_name, GetThunkKindName(thunk_kind));
-      return new_thread_plan_sp;
+      return nullptr;
```

Comment:
long term this should be a StringError and the logging should happen at a higher level?

Reply:
That would probably make sense to simplify the logging process!